src/fty_config_manager.cc

```diff
@@ -124,34 +134,50 @@ namespace config
             // Check if the command is implemented
             if (configQuery.action.compare(SAVE_ACTION) == 0)
             {
-                // Get the configuration file path name from class variable m_parameters
-                std::string configurationFileName = AUGEAS_FILES + m_parameters.at(responseDto.featureName) + ANY_NODES;
-                log_debug("Configuration file name '%s'", configurationFileName.c_str());
-                cxxtools::SerializationInfo si;
-                getConfigurationToJson(si, configurationFileName);
+                std::map<std::string, cxxtools::SerializationInfo> configSiList;
+                std::vector<std::string> featureList = splitString(configQuery.featureName);
```

Comment:
Instead of splitting strings by hand, why not have directly an array of features in JSON?

Reply:
done